packages/config/config/devices/0x0371/zwa039.json

```diff
@@ -274,6 +274,30 @@
 			"$import": "~/0x0086/templates/aeotec_template.json#celsius_fahrenheit",
 			"valueSize": 1
 		},
+		{
+			"#": "92",
+			"$if": "firmwareVersion >= 2.1",
+			"label": "Battery Calibration Check",
+			"description": "To determine the battery calibration value, you must set each step in order",
+			"unsigned": true,
+			"valueSize": 4,
+			"allowManualEntry": false,
+			"defaultValue": 83951616,
```

Comment:
I'm not sure if the default value is correct. Also if I understood your email correctly, this can return values outside the specified range. So we should definitely add
```jsonc
			"minValue": 0,
			"maxValue": 4294967295
```
since the values are inferred from the options now.